frontend/src/common/components/table/options/Trash.tsx

```diff
@@ -2,21 +2,30 @@ import { IconButton } from "@mui/material";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faTrashCan } from "@fortawesome/free-solid-svg-icons";
 
+import "./Trash.scss";
+
 export const Trash = ({
   onClickTrash,
+  disabled,
 }: {
   /**
    * A callback function on clicking the trash icon.
    * @returns void
    */
   onClickTrash: () => void;
+  /**
+   * Boolean value indicating whether or not the trash button is disabled.
+   */
+  disabled?: boolean;
```

Comment:
suggest removing the comment. It's objectively pretty clear to understand the context of `disabled` IMHO